Client/src/Components/navbar.jsx

```diff
@@ -1,19 +1,184 @@
+  right: 40%;
+  height: 11vw;
+  width: auto;
+  z-index: 0; 
+  @media (max-width: 768px) {
+    display: none;
+  }
+`;
+
+const DesktopMenu = styled.div`
+  display: none;
+  @media (min-width: 768px) {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+`;
+
+const NavLink = styled(RouterNavLink)`
+  color: white;
+  text-decoration: none;
+  position: relative;
+  padding-bottom: 0.25rem;
+  transition: color 0.3s;
+
+  &:hover {
+    color: #60A5FA;
+  }
+
+  &.active {
+    color: white;
+    pointer-events: none;
+    cursor: default;
+  }
+
+  &.active::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    bottom: -2px;
+    height: 4px;
+    border-radius: 25px;
+    width: 100%;
+    background-color: #3B82F6;
+  }
+`;
+
+const ApplyButton = styled.a`
+  background-color: #fff;
+  color: #3B82F6;
+  font-weight: bold;
+  border: 2px solid #3B82F6;
+  padding: 0.5rem 0.75rem;
+  border-radius: 0.5rem;
+  text-decoration: none;
+  transition: background-color 0.3s;
+
+  &:hover {
+    background-color: #3B82F6;
+    color: #fff;
+  }
+`;
+
+const MobileMenuButton = styled.button`
+  color: white;
+  background: none;
+  border: none;
+  cursor: pointer;
+  display: block;
+  font-size: 1.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: transform 0.2s ease-in-out;
+
+  &:hover {
+    transform: scale(1.1);
+  }
+
+  @media (min-width: 768px) {
+    display: none;
+  }
+`;
+
+const MobileMenu = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding-top: 1rem;
+  @media (min-width: 768px) {
+    display: none;
+  }
+`;
 
 function Navbar() {
+  const [isOpen, setIsOpen] = useState(false);
+
   return (
-    <div>
-      Create Navbar component
-    </div>
-  )
+    <NavContainer>
+      <NavContent>
+        <RouterNavLink to="/">
+          <Logo src={logo} alt="HACKCUNY Logo" />
+        </RouterNavLink>
+        <RailsLogo src={railsLogo} alt="NYC Rails" />
+        <DesktopMenu>
+          <NavLink to="/" exact>
+            Home
+          </NavLink>
+          <NavLink to="/schedule">
+            Schedule
+          </NavLink>
+          <NavLink to="/team">
+            Our Team
+          </NavLink>
+          <NavLink to="/sponsors">
+            Sponsors
+          </NavLink>
+          <NavLink to="/faqs">
+            FAQS
+          </NavLink>
+          <ApplyButton href="/apply">Apply</ApplyButton>
+        </DesktopMenu>
+        <MobileMenuButton onClick={() => setIsOpen(!isOpen)} aria-label="Toggle menu">
+          {isOpen ? '✕' : '☰'}
+        </MobileMenuButton>
+      </NavContent>
+      {isOpen && (
+        <MobileMenu>
+          <NavLink to="/" exact onClick={() => setIsOpen(false)}>
+            Home
+          </NavLink>
+          <NavLink to="/schedule" onClick={() => setIsOpen(false)}>
+            Schedule
+          </NavLink>
+          <NavLink to="/team" onClick={() => setIsOpen(false)}>
```

Comment:
Good work, only change here is it should be "/our_team" for Team page url. @momoperv123

Reply:
oh, and same for the mobile view "/team" link to "/our_team"